src/Client.js

```diff
@@ -61,16 +61,38 @@ class Client extends EventEmitter {
         Util.setFfmpegPath(this.options.ffmpegPath);
     }
 
+    /**
+     * This function can be called to start a WS connection
+     * options:{
+     *     ...
+     *     puppeteer: { 
+     *         browserWSEndpoint: `ws://localhost:5001`
+     *         ...
+     *    }
+     * }
+     */
+    async connect(){
+
+        try {
+            this.pupBrowser = await puppeteer.connect(this.options.puppeteer);
+        } catch (err) {
+            this.emit(Events.ENOTFOUND, err.message);
+            return false;
+        }
+        await this.initialize();
+    }
```

Comment:
The way you handle the error here is not appropriate.. Maybe check if the client has been provided a browserWSEndpoint first?